cluster/examples/kubernetes/minio/operator.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook-minio-system
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectstores.minio.rook.io
+spec:
+  group: minio.rook.io
+  names:
+    kind: ObjectStore
+    listKind: ObjectStoreList
+    plural: objectstores
+    singular: objectstore
```

Comment:
we could consider a short name for this as well, similar to https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/operator.yaml#L49